tomcat-manager/Dockerfile

```diff
@@ -7,13 +7,18 @@ ARG APIMAN_VERSION
 
 USER root
 ADD https://github.com/Apiman/apiman/releases/download/$APIMAN_VERSION/apiman-distro-tomcat-$APIMAN_VERSION-overlay.zip $CATALINA_HOME
-RUN groupadd -r tomcat && useradd --no-log-init -r -g tomcat tomcat
+RUN groupadd -r tomcat && \
+    useradd --no-log-init --create-home -r -g tomcat tomcat && \
+    chown -R tomcat:tomcat /home/tomcat && \
+    apt-get update && \
+    apt-get install -y unzip && \
+    apt-get clean
 
 WORKDIR $CATALINA_HOME
-RUN apt-get update && \
-    apt-get install -y unzip && \
-    apt-get clean && \
-    unzip -o apiman-distro-tomcat-$APIMAN_VERSION-overlay.zip && \
+RUN unzip -o apiman-distro-tomcat-$APIMAN_VERSION-overlay.zip && \
+    # speed up tomcat
+    unzip -o webapps/apimanui.war -d webapps/apimanui && \
```

Comment:
You are using exploded WAR files here, I take it?

Reply:
Yes, the default war files from our tomcat overlay in the main project :)
It makes the startup a bit faster and also helps you to edit the files directly in a startup script (if needed).

Will end up in 
```bash
2021-08-17T08:54:30.3569518Z Archive:  webapps/apiman.war
2021-08-17T08:54:30.3601079Z   inflating: webapps/apiman/META-INF/MANIFEST.MF  
2021-08-17T08:54:30.3601891Z    creating: webapps/apiman/WEB-INF/
2021-08-17T08:54:30.3602630Z    creating: webapps/apiman/WEB-INF/classes/
2021-08-17T08:54:30.3603406Z    creating: webapps/apiman/WEB-INF/classes/io/
2021-08-17T08:54:30.3604201Z    creating: webapps/apiman/WEB-INF/classes/io/apiman/
2021-08-17T08:54:30.3605244Z    creating: webapps/apiman/WEB-INF/classes/io/apiman/manager/
2021-08-17T08:54:30.3607094Z    creating: webapps/apiman/WEB-INF/classes/io/apiman/manager/api/
2021-08-17T08:54:30.3608258Z    creating: webapps/apiman/WEB-INF/classes/io/apiman/manager/api/war/
2021-08-17T08:54:30.3609306Z    creating: webapps/apiman/WEB-INF/classes/io/apiman/manager/api/war/tomcat8/
2021-08-17T08:54:30.3610338Z    creating: webapps/apiman/WEB-INF/classes/META-INF/
2021-08-17T08:54:30.3611269Z    creating: webapps/apiman/WEB-INF/lib/
2021-08-17T08:54:30.3612041Z    creating: webapps/apiman/META-INF/maven/
2021-08-17T08:54:30.3612898Z    creating: webapps/apiman/META-INF/maven/io.apiman/
2021-08-17T08:54:30.3613968Z    creating: webapps/apiman/META-INF/maven/io.apiman/apiman-manager-api-war/
2021-08-17T08:54:30.3615620Z   inflating: webapps/apiman/WEB-INF/classes/io/apiman/manager/api/war/TransactionWatchdogFilter.class  
2021-08-17T08:54:30.3617250Z   inflating: webapps/apiman/WEB-INF/classes/io/apiman/manager/api/war/tomcat8/Tomcat8ApiManagerApplication.class  
2021-08-17T08:54:30.3618836Z   inflating: webapps/apiman/WEB-INF/classes/io/apiman/manager/api/war/tomcat8/Tomcat8PluginRegistry.class  
2021-08-17T08:54:30.3620145Z   inflating: webapps/apiman/WEB-INF/classes/io/apiman/manager/api/war/WarCdiFactory$1.class  
2021-08-17T08:54:30.3621469Z   inflating: webapps/apiman/WEB-INF/classes/io/apiman/manager/api/war/WarCdiFactory.class  
2021-08-17T08:54:30.3623104Z   inflating: webapps/apiman/WEB-INF/classes/io/apiman/manager/api/war/WarApiManagerBootstrapperServlet.class  
2021-08-17T08:54:30.3625004Z   inflating: webapps/apiman/WEB-INF/classes/io/apiman/manager/api/war/WarApiManagerConfig.class  
2021-08-17T08:54:30.3626954Z   inflating: webapps/apiman/WEB-INF/classes/log4j2.xml  
...
```

Reply:
Fine by me 👍.

We could do this for all of our overlays.